### DISCUSSION TITLE
Hugging Token needed?

### DISCUSSION THREAD

**Discussion by benoib** [TheLastBen/fast-stable-diffusion#2721] [Q&A] [answered]
Hey guys!

I'm playing with the runpod script for stable diffusion xl.

Something is bothering me.

In the training script we have for instance

```
    # Load the tokenizers
    tokenizer_one = AutoTokenizer.from_pretrained(
        args.pretrained_model_name_or_path,
        subfolder="tokenizer",
        use_fast=False,
        use_auth_token=True,
    )
```
    
The use_auth_token= True implies that we should use a HuggingFace token. I can't find any anywhere in the Runpod script. How does it work?

When I try to run the example script on my own, I get this error: 

`huggingface_hub.utils._headers.LocalTokenNotFoundError: Token is required (`token=True`),
`
Which seems normal.

Where is the catch / what am I missing here? Obviously I tried running the training with

` --hub-token MY_TOKEN` passed as a parameter. Did not help either so I am a bit lost here

Thanks for your help,

Ben

**Comment by TheLastBen** [comment, marked as answer]
nothing missing, the XL repo is public so no need for a token

**Reply by benoib** [reply]
Yes I figured, But i get the error on the AutoTokenizer. So you mean that i have another problem with my args?
Thanks for the answer!

**Reply by TheLastBen** [reply]
don't add any arg
